### PR TITLE
add jSuites toolbar setting for HTML type column

### PR DESCRIPTION
**Pull request by VioletKnight7**
```
var data = [
    ['Jazz', 'Honda', '2019-02-12', '', true, '$ 2.000,00', '#777700'],
    ['Civic', 'Honda', '2018-07-11', '', true, '$ 4.000,01', '#007777'],
];

var jSuiteToolbar = [
        {
            content: 'undo',
            onclick: function() {
                document.execCommand('undo');
            }
        },
        {
            content: 'redo',
            onclick: function() {
                document.execCommand('redo');
            }
        },
        {
            type:'divisor'
        },
        {
            type:'select',
            width: '80px',
            options: [ 'Verdana', 'Arial', 'Courier New' ],
            render: function(e) {
                return '<span style="font-family:' + e + '">' + e + '</span>';
            },
            onchange: function(a,b,c,d) {
                document.execCommand('fontName', false, d);
            }
        },
        {
            type: 'select',
            width: '48px',
            content: 'format_size',
            options: ['X-small','Small','Medium','Large','X-large'],
            render: function(e) {
                return '<span style="font-size:' + e + '">' + e + '</span>';
            },
            onchange: function(a,b,c,d,e) {
                document.execCommand('fontSize', false, e+1);
            }
        }
    ];

jexcel(document.getElementById('spreadsheet'), {
    data:data,
    columns: [
        { type: 'html', title:'Car', width:120, toolbar: jSuiteToolbar },
        { type: 'dropdown', title:'Make', width:200, source:[ "Alfa Romeo", "Audi", "Bmw" ] },
        { type: 'calendar', title:'Available', width:200 },
        { type: 'image', title:'Photo', width:120 },
        { type: 'checkbox', title:'Stock', width:80 },
        { type: 'numeric', title:'Price', width:100, mask:'$ #.##,00', decimal:',' },
        { type: 'color', width:100, render:'square', }
     ]
});
```

### CODE REVIEW

File: src/js/jexcel.core.js [hunk 1]
```diff
@@ -1855,6 +1862,7 @@ console.log(ret);
                     jSuites.editor(div, {
                         focus: true,
                         value: value,
+                        toolbar: toolbar,
```
Comment (VioletKnight7) [inline src/js/jexcel.core.js:1865]: here is the jSuites' toolbar setting

**Review by geektcp — COMMENTED** [review]
ok

**Comment by pphod** [comment]
Hi Violet,

Thanks for that. But, for extended options to columns, we should use the options, because each of the editors has plenty of specific initialization properties. So, because of that, this PR is not necessary.


jexcel(document.getElementById('spreadsheet'), {
    data:data,
    columns: [
        { type: 'html', title:'Car', width:120, options: { toolbar: jSuiteToolbar } },
        { type: 'dropdown', title:'Make', width:200, source:[ "Alfa Romeo", "Audi", "Bmw" ] },
        { type: 'calendar', title:'Available', width:200 },
        { type: 'image', title:'Photo', width:120 },
        { type: 'checkbox', title:'Stock', width:80 },
        { type: 'numeric', title:'Price', width:100, mask:'$ #.##,00', decimal:',' },
        { type: 'color', width:100, render:'square', }
     ]
});

**Comment by VioletKnight7** [comment]
Thanks for replying to this, Paul. Your solution is much better. Go for it!

**Pull request closed without merging**